.travis.yml

```diff
@@ -13,9 +13,11 @@ before_script:
 - ./node_modules/.bin/eslint assets/js/components/*.js
 - ./node_modules/.bin/eslint assets/js/constants/*.js
 - # ./node_modules/.bin/eslint assets/js/controllers/*.js
+- ./node_modules/.bin/eslint assets/js/controllers/e*.js
 - # ./node_modules/.bin/eslint assets/js/core/*.js
 - # ./node_modules/.bin/eslint assets/js/directives/*.js
 - ./node_modules/.bin/eslint assets/js/filters/*.js
+- ./node_modules/.bin/eslint assets/js/helpers/*.js
```

Comment:
I will rename that. It is not really an angular concept, but thought it would be interesting to introduce this distinction from services, as helpers are not aware of business logic.